ONSdigital/design-system
# CTA for IA Treejack study

### What is the context of this PR?
- Used the phase banner to page template to provide a call to action to the IA treejack study.  
- Updated the phase banner component to make use of the badge optional
- Changed the phase banner content to align vertically top, similar to warning panel
- Addressed [a wrapping issue with the badge text](https://github.com/ONSdigital/design-system/discussions/8#discussioncomment-1041151)
- Added an alpha phase example to address [a comment left in the discussions forum](https://github.com/ONSdigital/design-system/discussions/8#discussioncomment-177824)

### How to review
- Check banner content and link, and ensure it's displayed across all pages
- Check alignment of phase banner content across viewports
